src/app/search/components/filter-items/FilterItems.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useContext } from "react";
+import { FiltersContext } from "@/app/search/providers/FiltersProvider";
+
+import Card from "@/components/card/Card";
+import { FiltersType } from "@/types/filters-type";
+import { SelectOptionType } from "@/types/select-option-type";
+
+import styles from "./FilterItems.module.css";
+
+export enum FILTER_VARIANT {
+  list = "list",
+  radio = "radio",
+}
+
+type Props = {
+  filterVariant: FILTER_VARIANT;
+  title: string;
+  options: SelectOptionType[];
+  fieldName: keyof FiltersType;
+};
+
+function FilterItems({ filterVariant, title, options, fieldName }: Props) {
+  const { filters, dispatchFilters } = useContext(FiltersContext);
+
+  const filter = (value: string, filterType: keyof FiltersType): void => {
+    dispatchFilters({
+      type: "filtered",
+      key: filterType,
+      value: value,
+    });
+  };
+
+  return (
+    <div className={styles["filter-items"]}>
+      <Card title={title} className={styles.items}>
```

Comment:
چه نیازی به این div اضافه‌ست؟
```suggestion
    <Card title={title} className={styles["filter-items"]}>
```